Backpack/Button/Classes/BPKButton.m

```diff
@@ -154,7 +154,7 @@ - (void)setup {
     self.spinner.translatesAutoresizingMaskIntoConstraints = NO;
     [self addSubview:self.spinner];
 
-    self.heightConstraint = [self.heightAnchor constraintEqualToConstant:[self heightForSize:self.size]];
+    self.heightConstraint = [self.heightAnchor constraintGreaterThanOrEqualToConstant:[self heightForSize:self.size]];
```

Comment:
would this make it grow infinitely if a this is the last constraint to be priotitized? 
like say I have:

BPKText - height based on text
  ||
BPKText - height based on text
  ||
BPKButton - potentially super tall?